src/components/portal/portal.tsx

```diff
@@ -11,10 +12,11 @@ export class Portal extends React.PureComponent<PortalProps> {
 
     public render() {
         const {children, ...rest} = this.props;
+        const uniqueId = globalId.getRootId(this);
         this.portalContent = (
-            <div {...rest} data-automation-id="PORTAL">{children}</div>
+            <div {...rest} data-automation-id={uniqueId}>{children}</div>
         );
-        return null;
+        return <span data-id={uniqueId} data-automation-id="ID_SPAN" style={{display: 'none'}}/>;
```

Comment:
`ID_SPAN` feels too generic. "SPAN" is not really interesting. It should also mention that it's reference to a portal part. Suggestion: "PORTAL_REF"

Reply:
agreed. i thought it hints about the way we implemented this and therefore easier to understand, but `PORTAL_REF` is also clear. will change